autoscheduler/scraper/tests/commands/scrape_courses_tests.py

```diff
@@ -105,6 +113,65 @@ def test_parse_meeting_does_save_model_correct(self):
                             start_time=begin_time, end_time=end_time,
                             meeting_type=meeting_type, section=section)
 
+    def test_parse_meeting_unescapes_building(self):
+        """ Tests that parse meeting converts escaped HTML characters like "&amp;" into
+        their actual buildings

```

Comment:
```suggestion
        """ Tests that parse meeting converts escaped HTML characters like "&amp;" into
            their actual buildings
```

Need to indent this correctly (squash into prev commit)